Develop

harshasp009 opened the pull request:
Hi,

        Please check the PR for latest changes made in jekyll site.


Thank You,
S.P.Sriharsha

Commit:
MKTG-2288-fixed the 404 page design

Commit:
MKTG-2289-optimize the images

Commit:
MKTG-2290-fixed the lower case issue

Commit:
made changes to that we can test the build process.

sichvoge commented:
Hi SP, can you quickly outline what fixes are in this PR? Would like to understand what to test for.

harshasp009 commented:
Hi,

      please check the blow changes made : 

         1. Added the 404 page as per Matt M request (https://www.mulesoft.org/jira/browse/MKTG-2288)
                 url: http://ramldev.debugme.in/404

         2.  added the optimize the images provided by the Matt M 
                   https://www.mulesoft.org/jira/browse/MKTG-2289 

         3. changed the blog and tag links to lower case as per Matt M request
                   https://www.mulesoft.org/jira/browse/MKTG-2290

Thank you,
S.P.Sriharsha

spsriharsha009 committed:
Create README.md

add the piece of line in local testing section

Commit:
MKTG-2262-fixed the jekyll issue

Commit:
Merge branch 'develop' of https://github.com/raml-org/raml-org into develop

Commit:
MKTG-2262-fixed the page layout link

Commit:
MKTG-2262-fixed the page case sensitive issue link

sichvoge committed:
changed tag

sichvoge reviewed (approved):
LGTM 👍
